modules/core/src/main/scala/org/tessellation/modules/HttpApi.scala

```diff
@@ -104,7 +104,8 @@ sealed abstract class HttpApi[F[_]: Async: SecurityProvider: KryoSerializer: Met
                 gossipRoutes.p2pRoutes <+>
                 trustRoutes.p2pRoutes <+>
                 globalSnapshotRoutes.p2pRoutes <+>
-                healthcheckP2PRoutes
+                healthcheckP2PRoutes <+>
+                services.consensus.p2pRoutes
```

Comment:
We don't want to follow the convention of creating routes outside of the service?